math/module_argument_cplexnum.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2020 @author PaulinaQuintero
+ * @file
+ *
+ * Program to calculate the modulus and argument of a complex number
```

Comment:
please add `\brief` in front of the line
`\brief Program to calculate the modulus and argument of a complex number`